spec/controllers/alchemy/admin/pages_controller_spec.rb

```diff
@@ -598,6 +598,32 @@ module Alchemy
             alchemy_get :edit, id: page.id
           end
         end
+
+        context 'if page is scoped' do
+          context 'to a single role' do
+            context 'user role does not match' do
+              before do
+                expect_any_instance_of(Page).to receive(:editable_by?).at_least(:once) { false }
+              end
+
+              it 'redirects to sitemap' do
+                alchemy_get :edit, id: page.id
+                expect(response).to redirect_to(admin_dashboard_path)
```

Comment:
This is strange. In the controller it says to redirect to `admin_pages_path`, but here you're testing for `admin_dashboard_path`. Where is it supposed to go, and where is it actually going? Why is this spec green?

IMO `admin_pages_path` is the better place to redirect to.